CONTRIBUTING.md

```diff
@@ -0,0 +1,856 @@
+# Bug reports
+
+Use the [issue tracker](https://github.com/lhorie/mithril.js/issues). Do check to make sure your bug hasn't already been filed. Please give the following information, where possible:
+
+1. The version of Mithril you're using, whether it's the dev version or [the version on npm](http://npm.im/mithril.js). The version on npm may not have all the latest bug fixes, so your bug might very well be fixed in the dev version.
```

Comment:
I'll just specify "and the browser name and version".

I can revert the comma change if you'd prefer, too. But to be honest, we can safely stop supporting IE 8 and earlier:

- We already use several ES5 features all over the code base. IE9 only doesn't support strict mode, but Mithril itself doesn't use any strict mode-specific features.
- Microsoft is going to end all updates, including security updates, for anything before IE9 on both desktop and server starting January 12th of next year.

The rest of the browsers pre-ES5 barely have a market share, though.